src/shared/lib/oql/oqlfilter.ts

```diff
@@ -1,21 +1,111 @@
 /* eslint camelcase: "off" */
```

Comment:
@adamabeshouse what's the main change here - adding types (js -> ts)? Anything else?

Reply:
that plus just the logic change I just indicated below